__tests__/app.test.js

```diff
@@ -57,6 +57,149 @@ describe('ENDPOINT TESTS', () => {
+                test("sort by votes", () => {
+                    return request(app)
+                        .get("/api/articles/?sort_by=votes")
+                        .expect(200)
+                        .then((res) => {
+                            expect(res.body.articles.length).toBeGreaterThan(0)
+                            expect(res.body.articles).toBeSortedBy("votes", {
+                                descending: true,
+                            });
+                        });
+                });
+                test("sort by comment_count", () => {
+                    return request(app)
+                        .get("/api/articles/?sort_by=comment_count")
+                        .expect(200)
+                        .then((res) => {
+                            expect(res.body.articles.length).toBeGreaterThan(0)
+                            expect(res.body.articles).toBeSortedBy("comment_count", {
+                                descending: true, coerce: true
+                            });
+                        });
+                });
+            });
+            describe('GET 200 - /api/articles/ - order', () => {
+                test("order ascending - sorted by default - created_at", () => {
+                    return request(app)
+                        .get("/api/articles/?order=asc")
+                        .expect(200)
+                        .then((res) => {
+                            expect(res.body.articles.length).toBeGreaterThan(0)
+                            expect(res.body.articles).toBeSortedBy("created_at", {
+                                ascending: true,
+                            });
+                        });
+                })
+                test("order ascending - sorted_by votes", () => {
+                    return request(app)
+                        .get("/api/articles/?sort_by=votes&order=asc")
+                        .expect(200)
+                        .then((res) => {
+                            expect(res.body.articles.length).toBeGreaterThan(0)
+                            expect(res.body.articles).toBeSortedBy("votes", {
+                                ascending: true,
+                            });
+                        });
+                })
+
+            });
+            describe('GET 200 - /api/articles/ - topic', () => {
+                test('filter by topic returning all topics', () => {
+                    return request(app)
+                        .get("/api/articles/?topic=mitch&sort_by=votes&order=asc")
+                        .expect(200)
+                        .then((res) => {
+                            expect(res.body.articles).toBeInstanceOf(Array)
+                            expect(res.body.articles.length).toBeGreaterThan(0)
+                            expect(res.body.articles).toBeSortedBy("votes", {
+                                ascending: true,
+                            });
+                        });
```

Comment:
this test currently isn't checking that the returned objects all have the topic of 'mitch'